docs/reference/content/driver/getting-started/installation-guide.md

```diff
@@ -0,0 +1,31 @@
++++
+date = "2015-03-17T15:36:56Z"
+title = "Installation Guide"
+[menu.main]
+  parent = "Sync Getting Started"
+  identifier = "Sync Installation Guide"
+  weight = 1
+  pre = "<i class='fa'></i>"
++++
+
+# Installation
+
+There are two different MongoDB drivers available in the 3.0 release.  The preferred method is to use the new MongoDB Driver as a 
```

Comment:
Just reading this again, it's confusing.  Especially now that we talk about two drivers: driver and async driver.

How about:

There are two Maven artifacts available in the 3.0 release.  The preferred artifact for new applications is mongodb-driver; however, we still publish the legacy mongo-java-driver uber-jar.